packages/core/src/utils.js

```diff
@@ -461,6 +461,27 @@ export function snapshotLogName(name, meta) {
   return name;
 }
 
+// parse browser cookies in correct format if flag is enabled
+// it assumes that cookiesStr is string returned by document.cookie
+export function parseCookies(cookiesStr) {
+  if (
+    process.env.PERCY_DO_NOT_USE_CAPTURED_COOKIES === 'true' ||
+    !(typeof cookiesStr === 'string' && cookiesStr !== '')
+  ) return null;
+
+  return cookiesStr.split('; ').map(c => {
+    const eqIdx = c.indexOf('=');
+    const cookieObj = {
+      name: c.substring(0, eqIdx),
+      value: c.substring(eqIdx + 1)
+    };
+    if (process.env.PERCY_SECURE_COOKIES === 'true') {
+      cookieObj.secure = true;
```

Comment:
Why is this required? also we cant capture secure cookies, how can we set those here?

Reply:
It is because for one of the customer setting  captured cookie was throwing error. I debugged it and found this. setting the cookie to secure solves the issue.
I have added support for this through env variable so it does not affect other cookie.

Reply:
Wouldnt it affect all the cookies ? All will become secure. And secure cookies. Ideally we should set these to same as in test browser. 

Please add a comment on this that new cookie capture process should update this code.

Reply:
reverted my changes. And i will be sending it in separate pr.